doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -149,6 +149,16 @@ nrf9160 Samples
       * An option ``--interval`` (in seconds) to neighbor cell measurements in continuous mode  (``link ncellmeas --continuous``).
         When using this option, a new measurement is started in each interval.
 
+Zigbee samples
+--------------
+
+* Updated documentation with newly introduced factory reset functionality in:
+  * :ref:`zigbee_light_bulb_sample`
```

Comment:
Sorry, one thing I just noticed in the doc build,  the formatting here seems to not be correct.

I believe you need to add a blank link just before this one so that it shows as a nested, bullet list.